Yoast/YoastSEO.js
# Add mobile preview

## Summary

This PR can be summarized in the following changelog entry:

## Relevant technical choices:

* Added a module for handling the toggles

## Test instructions

This PR can be tested by following these steps:

* Open the browserified example page and see if you can switch preview
* Default should be the mobile preview
* When on the desktop preview when the viewport is made smaller then 640px, it should be automatically switched to the mobile preview (this depends on the size of the viewport, so won't work in WordPress at the moment). 

requires: https://github.com/Yoast/wordpress-seo/pull/6423 when testing in WordPress. This makes sure the mobile preview is responsive in WordPress.
